config/helmchart/templates/manager.yaml

```diff
@@ -17,13 +17,29 @@ spec:
     {{- end }}
       labels:
         {{- include "namespace-configuration-operator.selectorLabels" . | nindent 8 }}
+        control-plane: controller-manager
     spec:
       serviceAccountName: namespace-configuration-operator-controller-manager
```

Comment:
is the service account this: `namespace-configuration-operator-controller-manager` or this `namespace-configuration-controller-manager` above it is corrected to the latter

Reply:
The label "control-plane: controller-manager" is used by the ServiceMonitor. 
The service account "namespace-configuration-operator-controller-manager" is correct, needs to match the kustomization output.